### PR TITLE
styled emails

### LINKED ISSUE #326: Style emails

### PR DESCRIPTION
**Pull request by eric-ls**
fix #326 Added basic styling to emails

screen shot 2017-03-06 at 11 47 00 am

### CODE REVIEW

File: app/controllers/api/lessons_controller.rb [hunk 1]
```diff
@@ -16,11 +16,12 @@ def create
   def update
     lesson = Lesson.find params[:id]
     # TODO: If the update only changes price and not time, then we shouldnt' send a reschedule email, but a Lesson Price Changed email.
-    if params[:is_paid]
-      # Send payment email
-    elif params[:start_time] and params[:end_time]
```
Comment (eric-ls) [inline app/controllers/api/lessons_controller.rb]: @echang9 i realize why this wasn't working before. It's suppsed to be `elsif`, not `elif`. Do you think the order should be reverted to what it was before?

Comment (echang9) [inline app/controllers/api/lessons_controller.rb]: i think the ordering is fine! (doesn't really make too much difference)

File: app/views/forte_mailer/student_signup_notify_student.text.erb [hunk 2]
```diff
@@ -4,5 +4,5 @@ Thank you for completing your registration.
 
 Please review the information below for accuracy. Please email any corrections or questions to TODO: Insert email here.
```
Comment (echang9) [inline app/views/forte_mailer/student_signup_notify_student.text.erb]: oh, any reason why there's a TODO here?

File: app/views/forte_mailer/teacher_signup_notify_teacher.text.erb [hunk 3]
```diff
@@ -4,5 +4,5 @@ Thank you for completing your registration.
 
 Please review the information below for accuracy. Please email any corrections or questions to TODO: Insert email here.
```
Comment (echang9) [inline app/views/forte_mailer/teacher_signup_notify_teacher.text.erb]: same here!

File: app/views/forte_mailer/student_signup_notify_student.text.erb (hunk 2 above)
Comment (eric-ls) [inline app/views/forte_mailer/student_signup_notify_student.text.erb]: Idk, that was there before, which email should I use?

Comment (echang9) [inline app/views/forte_mailer/student_signup_notify_student.text.erb]: oh good point... hmm maybe just default it to fortemailer@gmail.com (but i think it should be one of the forte admins) email

Comment (eric-ls) [inline app/views/forte_mailer/student_signup_notify_student.text.erb]: I'm gonna use contact@forteacademy.org for now, which is their contact email on the current website

**Comment by echang9** [comment]
looks good 👍 . merge when you're ready!

**Pull request merged**